include/taskolib/Sequence.h

```diff
@@ -69,6 +69,21 @@ namespace task {
  * // insert an action step at index 1
  * seq.insert(seq.begin() + 1, Step{Step::type_action});
  * \endcode
+ *
+ * \section Step setup script
+ *
+ * Each step can have a setup script that is executed before the intrinsic script is
+ * called (invoked with Step::set_script(const std::string&)). You can call it with
+ * set_step_setup_script(std::string).
+ * The defined variables and functions in the setup are exported in each step and you can
```

Comment:
As a peculiarity of Lua, only *global* variables survive from one chunk to the next:
```suggestion
 * Global variables and functions defined in the setup script can be accessed ...
```